AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -8368,6 +8368,7 @@ privacy.madame.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent
 consent.ladbible.com,consent.gamingbible.co.uk,consent.unilad.co.uk#%#AG_onLoad(function(){if(window.location.href.includes("consentUUID")){var g=new MutationObserver(function(){var b=document.querySelector('button[title*="Accept" i]');b&&(g.disconnect(),b.click())});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},1E4)}});
 !
 jobthai.com#%#//scriptlet('set-local-storage-item', 'website-visit-date', '1')
+flaschenpost.de#%#AG_onLoad(function(){var a=new MutationObserver(function(){var b=$('button.fp_button_primary:contains("Geht klar")');b&&b.click()});a.observe(document,{childList:!0,subtree:!0});setTimeout(function(){a.disconnect()},1E4)});
```

Comment:
Yep, I have also found that the clicking rule is ok here, but you need additional check.
Anyway, now it's fixed.